.cirrus.yml

```diff
@@ -89,28 +89,42 @@ on_failure_googlecompute: &on_failure_googlecompute
 
 
 task:
+  name: 'Building VM Image: ${IMAGE_NAME}'
+  alias: ${IMAGE_NAME}
```

Comment:
It might be good if the task aliases had vmbuild- or such as a prefix, otherwise we'll just end up with conflicts vs containers?

Reply:
There are dash differences, if we add vmbuild or such we should set depends_on for each test task. Does it worth it?

For example, current task:
```yaml
  matrix:
    - name: 'Testing VM Image: ${IMAGE_NAME}'
      env:
        IMAGE_NAME: windows-ci-mingw64
      compute_engine_instance:
        image_project: $GCP_PROJECT
        image: family/pg-ci-${IMAGE_NAME}
        platform: windows

    - name: 'Testing Container Image: ${IMAGE_NAME}'
      env:
        IMAGE_NAME: windows_ci_mingw64
      windows_container:
        image: $GCP_REPO/${IMAGE_NAME}:latest

  depends_on:
    - ${IMAGE_NAME}
```

Reply:
> There are dash differences, if we add vmbuild or such we should set depends_on for each test task. Does it worth it?

That's not that big a difference to today, is it? You can still continue to use ${IMAGE_NAME}.